openpdf/src/main/java/com/lowagie/text/pdf/BarcodeDatamatrix.java

```diff
@@ -919,25 +921,25 @@ public int getWidth() {
      * @param width the width of the barcode
      */
     public void setWidth(int width) {
-        this.width = width;
+        dimensions.setWidth(width);
     }
 
     /**
      * Gets the whitespace border around the barcode.
      *
      * @return the whitespace border around the barcode
      */
-    public int getWs() {
-        return ws;
+    public int getBorder() {
+        return dimensions.getBorder();
     }
 
     /**
      * Sets the whitespace border around the barcode.
      *
-     * @param ws the whitespace border around the barcode
+     * @param border the whitespace border around the barcode
      */
-    public void setWs(int ws) {
-        this.ws = ws;
+    public void setBorder(int border) {
+        dimensions.setBorder(border);
     }
```

Comment:
Here is important, that public methods are not simply renamed, as some users may  be using it. I'll merge anyway and add the getWs() and setWs() methods back, which will call the get/setBorder() methods.